src/c_kzg_4844.c

```diff
@@ -1310,7 +1310,7 @@ C_KZG_RET compute_aggregate_kzg_proof(KZGProof *out,
 
 C_KZG_RET verify_aggregate_kzg_proof(bool *out,
                                      const Blob blobs[],
-                                     const KZGCommitment *expected_kzg_commitments,
+                                     const KZGCommitment expected_kzg_commitments[],
```

Comment:
Why was this reverted? Is it just to keep changes minimal?

Otherwise, [it's the same thing](https://c-faq.com/aryptr/aryptrparam.html). Ppersonally I prefer the pointer form and I'm a bit sad that c-kzg uses both forms.

Reply:
I made that change before I decided to split things up into multiple PRs. Reverted it because it because that PR was focussed on polynomials and didn't want to create confusion. I'm pretty sure I made this same change in #62.

Reply:
Makes sense!